core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -3096,6 +3116,18 @@ public int getPartitionPurgeListCapacity() {
             return o3PartitionPurgeListCapacity;
         }
 
+        @Override
+        public int getPreferencesParserCacheSizeLimit() {
+            // the json to be parsed comes from the http receive buffer
+            // so setting the max size to the same
```

Comment:
this is confusing, the setting must make a reference to the JSON as this cache is specifically for caching large JSON values.

Reply:
removed it in https://github.com/questdb/questdb/pull/5621/commits/7df6e1be9d2df9e7c85eac3b668fbe8a864004ce

not needed, because the parser does not need cache for this use case.

Reply:
also changed the processor to handle fragmentation while sending responses